README.md

```diff
@@ -1968,55 +1968,53 @@ Testing repositories in that way leads to fragile tests and then refactoring is
 
 ## Test fixtures
 
-| :x: **BAD:** |
-|:-------------|
+| :x: **GOOD:** |
+|:--------------|
 
 ```php
-final class InvalidTest extends TestCase
+final class GoodTest extends TestCase
 {
-    private Subscription $subscription;
+    private SubscriptionFactory $sut;
 
     public function setUp(): void
     {
-        $this->subscription = new Subscription(new \DateTimeImmutable());
-        $this->subscription->activate();
+        $this->sut = new SubscriptionFactory();
     }
 
     /**
      * @test
      */
-    public function suspending_an_active_subscription_with_cannot_suspend_new_policy_is_possible(): void
-    {
-        $result = $this->subscription->suspend(new CannotSuspendNewSubscriptionPolicy(), new \DateTimeImmutable());
-
-        self::assertTrue($result);
-    }
-
-    /**
-     * @test
-     */
-    public function suspending_an_active_subscription_with_cannot_suspend_expired_policy_is_possible(): void
+    public function it_creates_a_subscription_for_a_given_date_range(): void
     {
-        $result = $this->subscription->suspend(new CannotSuspendExpiredSubscriptionPolicy(), new \DateTimeImmutable());
+        $result = $this->sut->create(new \DateTimeImmutable(), new \DateTimeImmutable('now +1 year'));
 
-        self::assertTrue($result);
+        self::assertInstanceOf(Subscription::class, $result);
     }
 
     /**
      * @test
      */
-    public function suspending_a_new_subscription_with_cannot_suspend_new_policy_is_not_possible(): void
+    public function it_throws_an_exception_on_invalid_date_range(): void
     {
-        // Here we need to create a new subscription, it is not possible to change $this->subscription to a new subscription
+        $this->expectException(CreateSubscriptionException::class);
+        
+        $result = $this->sut->create(new \DateTimeImmutable('now -1 year'), new \DateTimeImmutable());
     }
 }
 ```
 
-| :white_check_mark: **GOOD:** |
-|:-----------------------------|
+| :exclamation: **WARNING:** |
+|:---------------------------|
 
+- Any configuration made inside `setUp` couples test together, and has impact on all tests.
+- It's better to avoid a shared state between tests and configure initial state accordingly to test method.
+- The best case for this will be testing stateless objects. Readability is worse compared to configuration made in the
```

Comment:
_The best case for this will be testing stateless objects._ 

The best case for **using the setUp** will be testing stateless objects. And maybe Readability as a separate bullet?